landing-page/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>CtD class-Exerc.3-Landing page</title>
+   <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <header id="showcase">
+    <h1>Welcome!</h1>
+    <h1 id="simbol">Gizmatic</h1>
+        <p>Founded in 1901 Gizmatic has been making the world's best gizmos.</p> 
+    <a href="www.gizmatic.com/about" class="button1">Read More</a> 
+    
+    <section id="section-a">
+    <a class="button2"
+       href="https://www.gizmatic.com/customer&service">Service</a>
+    <a class="button2" 
+       href="https://www.gizmatic.com/contact">Contact</a>
+    <a class="button2" 
+       href="https://www.gizmatic.com/week&sale">Week Sale!</a>
+    </section> 
+    
+  </header>
+  
+    <section id="section-b">
+       <div class="column1" >
+       <a class="buttom3" href="#">Basic</a>  
+        <p class="">Perfect for your first project.</p>
```

Comment:
Having an empty string as a class value on this element won't break anything, but if it doesn't need a class this attribute should be deleted.